templates/problem/voting-stats.html

```diff
@@ -0,0 +1,383 @@
+                        ctx.lineTo(box_plot_x[2], box_plot_box_y);
+                        ctx.stroke();
+                        ctx.beginPath();
+                        ctx.moveTo(box_plot_x[1], box_plot_box_y+box_plot_height);
+                        ctx.lineTo(box_plot_x[2], box_plot_box_y+box_plot_height);
+                        ctx.stroke();
+                        ctx.beginPath();
+                        ctx.moveTo(box_plot_x[1], box_plot_box_y);
+                        ctx.lineTo(box_plot_x[1], box_plot_box_y+box_plot_height);
+                        ctx.stroke();
+                    }
+                    if (render_box_plot[3]) {//right edge of box
+                        ctx.beginPath();
+                        ctx.moveTo(box_plot_x[3], box_plot_box_y);
+                        ctx.lineTo(box_plot_x[2], box_plot_box_y);
+                        ctx.stroke();
+                        ctx.beginPath();
+                        ctx.moveTo(box_plot_x[3], box_plot_box_y + box_plot_height);
+                        ctx.lineTo(box_plot_x[2], box_plot_box_y + box_plot_height);
+                        ctx.stroke();
+                        ctx.beginPath();
+                        ctx.moveTo(box_plot_x[3], box_plot_box_y);
+                        ctx.lineTo(box_plot_x[3], box_plot_box_y + box_plot_height);
+                        ctx.stroke();
+                    }
+                }
+            {% endif %}
+            //draw median line
+            const median_height = plot_height * (5 / 6);
+            const median_y = plot_padding + plot_height / 2 - median_height / 2;
+            const median_x = points_to_animated_x({{ median_vote }});
+            if (document.getElementById('id_show_median_box').checked) {
+                ctx.strokeStyle = '#ff0000';
+                ctx.lineWidth = 5;
+                line(median_x, median_y, median_x, median_y + median_height);
+            }
+            //draw mean line
+            const mean_y = median_y;
+            const mean_height = median_height;
+            const mean_x = points_to_animated_x({{ mean_vote }});
+            if (document.getElementById('id_show_mean_box').checked) {
+                ctx.strokeStyle = '#00cc00';
+                ctx.lineWidth = 5;
+                line(mean_x, mean_y, mean_x, mean_y + mean_height);
+            }
+
+            {% if can_vote %}
+                //draw the individual votes
+                if (document.getElementById('id_show_vote_box').checked) {
+                    //vote size settings
+                    const vote_height = plot_height * (2 / 3);
+                    const vote_width = 10;
+                    const vote_y = plot_padding + plot_height / 2 - vote_height / 2;
+                    //get the votes from context
+                    let votes = [];
+                    {% for vote in all_votes %}
+                        votes.push({{ vote }});
+                    {% endfor %}
```

Comment:
Others' opinions wanted.
```suggestion
                    let votes = [{% for vote in all_votes %}{{ vote }},{% endfor %}];
```